Gopkg.lock

```diff
@@ -439,56 +347,20 @@
     "unicode/bidi",
     "unicode/cldr",
     "unicode/norm",
-    "unicode/rangetable",
+    "unicode/rangetable"
   ]
-  pruneopts = "UT"
   revision = "f21a4dfb5e38f5895301dc265a8def02365cc3d0"
   version = "v0.3.0"
 
 [[projects]]
-  digest = "1:342378ac4dcb378a5448dd723f0784ae519383532f5e70ade24132c4c8693202"
   name = "gopkg.in/yaml.v2"
   packages = ["."]
-  pruneopts = "UT"
   revision = "5420a8b6744d3b0345ab293f6fcba19c978f1183"
   version = "v2.2.1"
 
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  input-imports = [
-    "github.com/DATA-DOG/go-sqlmock",
-    "github.com/blang/semver",
-    "github.com/buger/jsonparser",
-    "github.com/chasex/glog",
-    "github.com/coocood/freecache",
-    "github.com/erikstmartin/go-testdb",
-    "github.com/evanphx/json-patch",
-    "github.com/golang/glog",
-    "github.com/julienschmidt/httprouter",
-    "github.com/lib/pq",
-    "github.com/magiconair/properties/assert",
-    "github.com/mssola/user_agent",
-    "github.com/mxmCherry/openrtb",
-    "github.com/mxmCherry/openrtb/native/request",
-    "github.com/prebid/go-gdpr/consentconstants",
-    "github.com/prebid/go-gdpr/vendorconsent",
-    "github.com/prebid/go-gdpr/vendorlist",
-    "github.com/prometheus/client_golang/prometheus",
-    "github.com/prometheus/client_golang/prometheus/promhttp",
-    "github.com/prometheus/client_model/go",
-    "github.com/rcrowley/go-metrics",
-    "github.com/rs/cors",
-    "github.com/spf13/viper",
-    "github.com/stretchr/testify/assert",
-    "github.com/vrischmann/go-metrics-influxdb",
-    "github.com/xeipuuv/gojsonschema",
-    "github.com/yudai/gojsondiff",
-    "github.com/yudai/gojsondiff/formatter",
-    "golang.org/x/net/context/ctxhttp",
-    "golang.org/x/net/publicsuffix",
-    "golang.org/x/text/currency",
-    "gopkg.in/yaml.v2",
-  ]
+  inputs-digest = "a0e2407806ef6c4ceced76db7e4be315c779a5c74453770c9e3f28626c2f30f7"
```

Comment:
This looks like it's regressing back to an older `dep` output -- would you be able to use 0.5.0?